src/main/java/seedu/address/logic/commands/AddTransactionCommand.java

```diff
@@ -66,10 +62,16 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        Person selected = lastShownList.get(index.getZeroBased());
-        Transaction toAdd = new Transaction(description, amount, otherParty, date);
-        selected.getTransactions().add(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, Messages.format(selected)));
+        Person personToEdit = lastShownList.get(index.getZeroBased());
+        List<Transaction> transactions = new ArrayList<>();
+        transactions.add(toAdd);
```

Comment:
This code here instantiates a new list, so it "forgets" the existing list of transactions. Maybe we want to get the existing list of transactions instead?